lib/bamboo/adapters/send_grid_adapter.ex

```diff
@@ -5,6 +5,10 @@ defmodule Bamboo.SendGridAdapter do
   Use this adapter to send emails through SendGrid's API. Requires that an API
   key is set in the config.
 
+  If you would like to add a replyto header to your email, then simply pass it in
+  using the header property or put_header function like so:
+  put_header("reply-to", "foo@bar.com")
```

Comment:
If you put this one more line down and indent it 4 spaces it will be highlighted as code, which I think would look nice and draw the reader's attention